tests/infra/ccf.py

```diff
@@ -538,20 +538,20 @@ def wait_for_all_nodes_to_catch_up(self, primary, timeout=3):
         while time.time() < end_time:
             with primary.node_client() as c:
                 resp = c.get("getCommit")
```

Comment:
Is it worth renaming `getCommit`, and/or its response fields `commit` and `term` in this PR? Or do we think this is going to be replaced by a generic `status` RPC soon, so there's no point breaking the names of it now?

Reply:
I thought about it but it seemed best to do it separately because I'd like that to become the node status.